app/router.js

```diff
@@ -29,7 +29,22 @@ const Router = Ember.Router.extend({
         if (!this.get('disableResetScroll')) {
             window.scrollTo(0, 0);
         }
+    },
+
+    didTransition(){
```

Comment:
Combine this with the `didTransition` above (thanks Travis!)

Reply:
@aaxelb I think this happened when merged with develop